tests/transpiler/braket/test_coverage_from_braket.py

```diff
@@ -35,6 +35,13 @@ def is_package_installed(package_name: str) -> bool:
 
 graph = ConversionGraph(require_native=True)
 
+# TODO: Update Pyqasm with the definitions / fix errors of these gates
+GATES_TO_SKIP = {
+    # "GPi2",
+    "MS",
+    "PRx",
+}
```

Comment:
Make sure to turn this into a GitHub issue

Reply:
Only `MS` gate is unsupported now, rest all are passing. There were some bugs in their implementations.

Reply:
Issue : https://github.com/qBraid/qBraid/issues/847